--- submodules/externals/ffmpeg/configure	2015-02-13 09:45:17.766769717 +0100
+++ submodules/externals/ffmpeg/configure.new	2015-02-13 09:44:26.891856924 +0100
@@ -79,6 +79,7 @@
   --logfile=FILE           log tests and output to FILE [config.log]
   --disable-logging        do not log configure debug information
   --fatal-warnings         fail if any configure warning is generated
+  --disable-warnings       disable all compiler warnings
   --prefix=PREFIX          install in PREFIX [$prefix]
   --bindir=DIR             install binaries in DIR [PREFIX/bin]
   --datadir=DIR            install data files in DIR [PREFIX/share/ffmpeg]
@@ -1541,6 +1542,7 @@
     cross_compile
     debug
     extra_warnings
+    warnings
     logging
     lto
     optimizations
@@ -3987,6 +3989,10 @@
 fi
 enabled sdl && add_cflags $sdl_cflags && add_extralibs $sdl_libs
 
+if disabled warnings; then
+    add_cflags -w
+fi
+
 texi2html --help 2> /dev/null | grep -q 'init-file' && enable texi2html || disable texi2html
 makeinfo --version > /dev/null 2>&1 && enable makeinfo  || disable makeinfo
 perl --version > /dev/null 2>&1 && enable perl || disable perl
